### DISCUSSION TITLE
I cant build GLAD in IMGUI

### DISCUSSION THREAD

**Discussion by Minster23** [ocornut/imgui#8261] [New Users Build/Link/Run issues ONLY!]
**Hi**
i trying to compile/build imgui with glad and opengl but it always error

**CONSOLE:**
![image](https://github.com/user-attachments/assets/d72ca4e1-fd63-4216-ba5a-89c4da90a6dd)

**DIRECTORY**
![image](https://github.com/user-attachments/assets/1dbe804f-4842-4e0f-9810-8cfde6bb1b76)

**CMAKE**
![image](https://github.com/user-attachments/assets/3ce98744-7cb1-4206-a0c1-7d5fc02dc926)

**Comment by ocornut** [comment]
Hello,
Sorry but your question has nothing to do with dear imgui. It is a glad and cmake question. Closing this
